.github/workflows/main.yaml

```diff
@@ -37,20 +37,27 @@ jobs:
           uses: messense/maturin-action@v1
           with:
             target: x86_64
-            args: --release --out dist --no-sdist -m Cargo.toml
+            args: --release --out dist -m Cargo.toml
 
         - name: Build wheels - x86_64 - CPython
           if: "!startsWith(matrix.python-version, 'pypy')"
           uses: messense/maturin-action@v1
```

Comment:
You might need to update this tag to https://github.com/PyO3/maturin-action

Reply:
\+